### PR TITLE
Google Container Engine - GKE [1]

### PR DESCRIPTION
**Pull request by mihaitusa**
Added several beta operations.

### CODE REVIEW

**Comment by mihaitusa** [comment]
@gituser173  please review!

File: content/io/cloudslang/cloud_provider/google/gke/beta_create_cluster.sl [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+####################################################
+
+namespace: io.cloudslang.cloud_provider.google.gke
+
+operation:
+  name: beta_create_cluster
+  inputs:
+    - projectId
+    - zone
+    - jSonGoogleAuthPath
```
Comment (gituser173) [inline content/io/cloudslang/cloud_provider/google/gke/beta_create_cluster.sl]: JSON not jSon

File: test/io/cloudslang/cloud_provider/google/gke/test_list.sl [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+          - return_result
+          - response
+          - error_message
+          - response_body: return_result
+
+    - print_ListClusters:
+        do:
+          print.print_text:
+            - text: response
+
```
Comment (gituser173) [inline test/io/cloudslang/cloud_provider/google/gke/test_list.sl]: Maybe an assert here?

**Comment by oritstone** [comment]
@mihaitusa please first update from master so we can merge to master
Then you will need to do the changes according to the new DSL changes, which I see are not done yet

**Pull request closed without merging**